### ISSUE TITLE
Why net.ifnames=0 is used ?

### ISSUE THREAD

**Issue by Et7f3** [osbuild/images#620]
I digged the git history: I come to this commit (I didn't find a related PR so I open this issue). It seems to be added as an example of customization and then copied from version to version without thinking about it.

https://github.com/osbuild/images/commit/22254637f810f1cabc95fb616b46d92049135e99#diff-de240f0bca8f03daafeb653ff429aafa12f43b5e0dc8ba327c56108a6ff864f3R70

Original issue is I can't change it:

```
# grep -r ifname /etc
/etc/default/grub:GRUB_CMDLINE_LINUX="console=tty0 console=ttyS0,115200n8 no_timer_check net.ifnames=1"
grep: /etc/udev/hwdb.bin: binary file matches
/etc/kernel/cmdline:root=UUID=020df4af-98fd-4818-933d-321ce6635b45 console=tty0 console=ttyS0,115200n8 no_timer_check net.ifnames=1
```

regen grub config and reinstall kernel-core to trigger dracut regeneration.

**Comment by achilleas-k** [comment]
This will change in Fedora 41: https://fedoraproject.org/wiki/Changes/EnableConsistentDeviceNamingCloud

As for the history of the option: I can't really say with absolute certainty, but it was likely added to replicate the official Fedora cloud images.  [This comment](https://discussion.fedoraproject.org/t/f41-change-proposal-enable-consistent-device-naming-in-cloud-images-self-contained/108894/7) in the change proposal discussion thread links to the oldest surviving instance of the change being introduced.

**Comment by supakeen** [comment]
> This will change in Fedora 41: https://fedoraproject.org/wiki/Changes/EnableConsistentDeviceNamingCloud
> 
> As for the history of the option: I can't really say with absolute certainty, but it was likely added to replicate the official Fedora cloud images. [This comment](https://discussion.fedoraproject.org/t/f41-change-proposal-enable-consistent-device-naming-in-cloud-images-self-contained/108894/7) in the change proposal discussion thread links to the oldest surviving instance of the change being introduced in the official Fedora kickstarts.

Heh, the change proposal you linked says that current Fedora's except cloud use consistent device naming so it seems to confirm that our definitions would be wrong for anything not-cloud?

**Comment by achilleas-k** [comment]
Yes, I think the issue started by replicating cloud configs and then it carried over to non-cloud images.

**Comment by ondrejbudai** [comment]
@achilleas-k this is now fixed for Fedora and EL10, right?

**Comment by achilleas-k** [comment]
I never made it configurable but yes, this is done.
